terrain/voxel_lod_terrain.cpp

```diff
@@ -32,8 +32,8 @@ Ref<ArrayMesh> build_mesh(const Vector<Array> surfaces, Mesh::PrimitiveType prim
 			continue;
 		}
 
-		if (collidable_surface != nullptr && collidable_surface->empty()) {
-			*collidable_surface = surface;
+		if (collidable_surface[0]->empty()) {
+			*collidable_surface[0] = surface;
```

Comment:
And then this can be `collidable_surfaces != nullptr`. tbh I might do a second pass on this later because collision support may need some refinements beyond support for multiple surfaces